experiments/rtp-hdrext/inband-cn/index.md

```diff
@@ -0,0 +1,59 @@
+---
+
+**Name:** "Inband Comfort Noise" ; "RTP Header Extension to signal inband comfort noise"
+
+**Formal name:** <http://www.webrtc.org/experiments/rtp-hdrext/inband-cn>
+
+**Status:** This extension is defined here to allow for experimentation. Once experience has shown that it is useful, we intend to make a proposal based on it for standardization in the IETF.
+
+## Introduction
+
+Comfort noise (CN) is widely used in real time communication, as it significantly reduces the frequency of RTP packets, and thus saves the network bandwidth, when participants in the communication are constantly actively speaking.
+
+One way of deploying CN is through [RFC 3389]. It defines CN as a special payload, which needs to be encoded and decoded independently from the codec(s) applied to active speech signals. This deployment is referred to as outband CN in this context.
+
+Some codecs, for example, RFC 6716: Definition of the Opus Audio Codec, implement their own CN schemes. Basically, the encoder can notify that a CN packet is issued and/or no packet needs to be transmitted.
+
+Since CN packets have their particularities, cloud and client may need to identify them and treat them differently. Special treatments on CN packets include but are not limited to
+
+* Upon receiving multiple streams of CN packets, choose only one to relay or mix.
+* Adapt jitter buffer wisely according to the discontinuous transmission nature of CN packets.
+
+While RTP packets that contain outband CN can be easily identified as they bear a different payload type, inband CN cannot. Some codecs may be able to extract the information by decoding the packet, but that depends on codec implementation, not even mentioning that decoding packets is not always feasible. This document proposes using an RTP header extension to signal the inband CN. 
+
+## RTP header extension format
+
+The iband CN extension can be encoded using either the one-byte or two-byte header defined in [RFC 5285].  Figures 1 and 2 show encodings with each of these header formats.
+
+ 0                   1
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  ID   | len=0 |N| noise level |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ Figure 1. Encoding Using the One-Byte Header Format
+
+ 0                   1                   2
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|      ID       |     len=1     |N| noise level |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ Figure 2. Encoding Using the Two-Byte Header Format
+
+Noise level is an optional data. The bit "N" being 1 indicates that there is a noise level. The noise level is defined the same way as the audio level in [RFC 6464] and therefore can be used to avoid the Audio Level Header Extension on the same RTP packet. This also means that this level is defined the same as the noise level in [RFC 3389] and therfore can be compared against outband CN.
+
```

Comment:
The N bit seems unnecessary. Why not make the whole noise level 0 when no noise is present? That also gives you 8 bits to encode noise level rather than 7
(of course this is useless when using RFC 6464 definitions)

Reply:
The N bit is intended to make the audio level field optional. So one does not need to provide the noise level. 

This can also to make it live together with RFC 6464. Say if RFC 6464 is also in the RTP packet, one does not need to calculate audio level again for inband CN.

Do you have other suggestions?

Reply:
This seems good with explanation. Thanks!